### PR TITLE
Docs Guideline Proposal

### PR DESCRIPTION
**Pull request by Amstutz**
Proposal for naming, structuring and placing doc files (such as readmes) in ILIAS. I do not have a strong opinion on those proposals, so feel free to ask for, or even better push changes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Docs
+
+Docs contains information about the ILIAS license as well as useful help for installing
+and configuring ilias ([configuration](configuration)) and for understanding the code
```
Comment (chfsx) [inline docs/README.md]: ILIAS always in capitals! And pray to god Matthias does not see this :-)

File: docs/documentation/docs-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* Only the following files MUST be written in uppercase:
+  * README.md
+  * LICENSE.md
+* All other files MUST be written in lowercase.
+* Names containing more than one word MUST use hyphens to separate the words (such as
+docs-guidelines).
+* The file extension for all markdown files MUST be '.md'
+
+## Content
+* Markdown SHOULD be used for providing information such as described above.
```
Comment (chfsx) [inline docs/documentation/docs-guidelines.md:41]: I'd like a MUST here...

**Review by chfsx — CHANGES_REQUESTED** [review]
How about adding a link or a section which described how changes should be proposed? (PR agains trunk and label JourFixe).

Thankss a lot for your work!

File: docs/documentation/docs-guidelines.md (hunk 2 above)
Comment (gpkvt) [reply docs/documentation/docs-guidelines.md:41]: I second that. Markdown is easy readable as plain text, integrates well with Github and can be transformed into other formats without hassle.

**Comment by Amstutz** [comment]
@chfsx the requested section which describes how changes should be proposed has been added.

**Review by chfsx — APPROVED** [review]

**Comment by gpkvt** [comment]
How do we handle links to other docs which are available inside the ILIAS repo? In INSTALL.md I used relative paths, whichs works fine on github. When you read the document on you local machine after you checked out the repo it might be a bit unconvienient, but at least you know that the document is also part of the repository and available locally. 

Also there is no need to change the path for each release, as it would be if we use full URLs.

Any opinion on this?

**Comment by Amstutz** [comment]
@gpkvt Thank you for your for this remark. I agree, internal links seem to fit best for this. I added a respective rule.

File: docs/documentation/docs-guidelines.md (hunk 2 above)
Comment (matthiaskunkel) [reply docs/documentation/docs-guidelines.md:41]: Jour Fixe, April 10, 2017: Thank you for this PR. It can be merged.

**Pull request merged**